Core/MIPS/x86/CompVFPU.cpp

```diff
@@ -2169,7 +2209,7 @@ void Jit::Comp_VV2Op(MIPSOpcode op) {
 		} else {
 			MOV(32, R(EAX), fpr.V(sreg));
 		}
-		CallProtectedFunction((const void *)sinCosFunc, R(EAX));
+		CallProtectedFunction((const void *)sinCosFunc, R(EAX), Imm32((uint32_t)(uintptr_t)&mips_->sincostemp[0]));
```

Comment:
Why not just `R(RDX)`/`R(RDI)`?  Won't this fail if the pointer isn't in low RAM?  Isn't that the case on iOS or retroarch or something?

-[Unknown]